### PR TITLE
[BFCL] add MadeAgents/Hammer-7b handler

### PR DESCRIPTION
**Pull request by linqq9**
This PR add  MadeAgents/Hammer-7b.  Here's the CSV table converted to markdown format:

|   | Overall Acc |              Model             | AST Summary | Exec Summary | Simple AST | Multiple AST | Parallel AST | Parallel Multiple AST | Simple Exec | Multiple Exec | Parallel Exec | Parallel Multiple Exec | Irrelevance Detection | Relevance Detection |
|:-:|:-----------:|:------------------------------:|:-----------:|:------------:|:----------:|:------------:|:------------:|:---------------------:|:-----------:|:-------------:|:-------------:|:----------------------:|:---------------------:|:-------------------:|
| 1 |    83.92%   | Hammer-7b (fc) |    78.70%   |    89.71%    |   69.31%   |    82.52%    |    78.88%    |         84.08%        |    91.86%   |     94.00%    |     88.00%    |         85.00%         |         72.87%        |        92.68%       |

### CODE REVIEW

File: berkeley-function-call-leaderboard/bfcl/model_handler/hammer_handler.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+model_name1  = ''
+
+class hammerHandler(OSSHandler):
+    def __init__(
+        self, model_name, temperature=0.001, top_p=1, max_tokens=512, dtype="bfloat16"
+    ) -> None:
+        super().__init__(model_name, temperature, top_p, max_tokens, dtype)
+        self.model_style = ModelStyle.OSSMODEL
+        global model_name1
```
Comment (HuanzhiMao) [inline berkeley-function-call-leaderboard/bfcl/model_handler/hammer_handler.py:33]: What's the purpose of this `model_name1`?

Comment (linqq9) [reply berkeley-function-call-leaderboard/bfcl/model_handler/hammer_handler.py:33]: Sorry, model_name1 is redundant and I forgot to delete it. Can you help me delete it, or do I need to delete it and resubmit the pull request

Comment (linqq9) [reply berkeley-function-call-leaderboard/bfcl/model_handler/hammer_handler.py]: Hi, I have already fixed it. Thanks for the reminder.

**Review by HuanzhiMao — APPROVED** [review]
Thanks for the PR @linqq9! Excited to see the new models on the leaderboard.
LGTM.
I will simplify the code a bit and have the `HammerHandler` inherit from `xLAMHandler` since they share a lot of duplicate helper methods.

**Comment by HuanzhiMao** [comment]
Since I don't have permission to make edits to your PR branch directly, I have raised a PR to simplify the code logic. https://github.com/MadeAgents/gorilla/pull/1

**Comment by linqq9** [comment]
> Since I don't have permission to make edits to your PR branch directly, I have raised a PR to simplify the code logic. [MadeAgents#1](https://github.com/MadeAgents/gorilla/pull/1)

Thank you for your modification. I have completed the merge pull requests

**Pull request closed without merging**